### DISCUSSION TITLE
Pterodactyl

### DISCUSSION THREAD

**Discussion by KaleidoVR** [a-sync/game-server-watcher#9] [Q&A] [answered]
Any progress on getting the Egg done for Pterodactyl? :)

**Comment by a-sync** [comment]
not really 😓
but ill give it a go over the weekend!

something like this also might work: https://github.com/DEVBenSon/pterodactyl-eggs/blob/main/Eggs-Pack/%5B%F0%9F%A4%96%5D%20Bots/Discord%20Bots/discord.js/egg-discord-js-generic.json
__(but i did not test it, and you will need to setup a persistent storage for your /data/ folder)__

**Comment by a-sync** [comment, marked as answer]
Good new @ReiniVT!
I finally got around to finish and test the egg. It's available on the master branch, or directly here: https://raw.githubusercontent.com/a-sync/game-server-watcher/master/egg.json
